### PR TITLE
Simplify challenge wording

### LINKED ISSUE #24455: Rephrasing ES6: Use * to Import Everything from a File
**Describe your problem and how to reproduce it:**
The last 2 paragraphs of the exercise could use a rewording and some adjustments as they do not get the point across.

I.E. 

> The code below requires the contents of a file, "capitalize_strings", found in the same directory as it, imported.

The above line could use a refactoring of the statement for better readability.

> In order to utilize this method, it requires an object that receives the imported values. From here, you will use the dot notation to call your imported values.

Maybe consider saying, **In order to utilize this method, an object is required**

**Add a Link to the page with the problem:**
[Challenge](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file/)

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: Version 70.0.3538.67 (Official Build) (64-bit)
* Operating System: Windows 7 64bit

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
fccbug2

**If there is a possibility, after a clear rewording is selected, I would like to fix this issue.**

### PR DESCRIPTION
**Pull request by Lewis65**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #24455

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ challengeType: 1
...
-You may use any name following the <code>import * as </code>portion of the statement. In order to utilize this method, it requires an object that receives the imported values. From here, you will use the dot notation to call your imported values.
+You may use any name following the <code>import * as </code>portion of the statement. In order to utilize this method, it requires an object that receives the imported values (i.e., you must provide a name). From here, you will use the dot notation to call your imported values.
 </section>
 
 ## Instructions
 <section id='instructions'>
-The code below requires the contents of a file, <code>"capitalize_strings"</code>, found in the same directory as it, imported. Add the appropriate <code>import *</code> statement to the top of the file, using the object provided.
+The code in this file requires the contents of another file, <code>"capitalize_strings"</code>. <code>"capitalize_strings"</code> is in the same directory as the current file. Add the appropriate <code>import *</code> statement to the top of the file, using the object provided.
```
Comment (TomerPacific) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md]: Maybe change : 

> The code in this file requires the contents of another file, <code>"capitalize_strings"</code <code>"capitalize_strings"</code> is in the same directory as the current file.

To :  The code in this file requires the contents of another file, <code>"capitalize_strings"</code <code>"capitalize_strings"</code> that is located in the same directory.

Comment (Lewis65) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md]: Just changed it. Thanks!

**Review by TomerPacific — APPROVED** [review]

**Comment by nikrb** [comment]
tested this locally. The curriculum test passes but I couldn't get the app tests to pass.

pr25120appresults

Am I doing something wrong?

**Comment by nikrb** [comment]
after reading #28264 I think we need to change the test regexp and solution. We seem to be missing dot forward slash for the included file which is in the same directory. So the testString regexp needs changing to something like:
`import\s+\*\s+as\s+[a-zA-Z0-9_$]+\s+from\s*("|')\.\/capitalize_strings\1\s*;/g`
Then the solution would need fixing to:
`import * as capitalize_strings from "./capitalize_strings";`

@Lewis65 would you like to make these changes?

**Comment by Nirajn2311** [comment]
@Lewis65 Any updates as it has been a month?
@nikrb Should I close this PR?

**Comment by RandellDawson** [comment]
@Nirajn2311 I think the main question is for this PR, do the changes made based on the original suggestion by @TomerPacific improve the challenge at all?  If so, I say we merge this one and in my PR (#28264) we can further discuss the final fix for this challenge.  This way, @Lewis65 gets credit for an improvement (as long as you think it is an improvement).  We should probably edit the `Closes #####` in the PR body as more than likely the other PR will actually be the one to close out the original issue.

**Comment by thecodingaviator** [comment]
@RandellDawson I surely think that this PR makes the description better. As far as the changes recommended by @TomerPacific on the issue are concerned, they just make the challenge more confusing.

If you want to go ahead with making the final fixes in your PR, that is perfectly fine too.

**Review by RandellDawson — APPROVED** [review]
I say we go ahead and merge this one and deal with the regex in PR #28264.

**Review by Manish-Giri — APPROVED** [review]
@Lewis65 LGTM. Thank You for your contribution.

**Pull request merged**